File: pinyougou_manager_web/src/main/webapp/js/service/specificationService.js
app.service("specificationService",function ($http) {
    this.findAll = function () {
        return $http.get("../specification/findAll.do")
    }
    this.findPage = function (pageNum,pageSize) {
        return $http.get("../specification/findPage.do?pageNum="+pageNum+"&pageSize="+pageSize)
    }
    this.add = function (entity) {
        return $http.post("../specification/add.do",entity)
    }
    this.update = function (entity) {
        return $http.post("../specification/update.do",entity)
    }
    this.findOne = function (id) {
        return $http.get("../specification/findOne.do?id="+id)
    }
    this.dele = function (ids) {
        return $http.get("../specification/delete.do?ids="+ids)
    }
    this.search = function (searchEntity, pageNum, pageSize) {
        return $http.post("../specification/search.do?pageNum="+pageNum+"&pageSize="+pageSize,searchEntity)
    }
    this.selectSpecList = function () {
        return $http.get("../specification/selectSpecList.do")
    }
})

File: pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import entity.PageResult;
import groupEntity.Specification;
import com.pinyougou.pojo.TbSpecification;

import java.util.List;
import java.util.Map;

/**
 * @Author: Liuyu
 * @Date: 2018/12/5 0005 19:54
 * @Version 1.0
 * 键盘敲烂,月薪过万
 */
public interface SpecificationService {
    PageResult search(TbSpecification specification, Integer pageNum, Integer pageSize);

    void add(Specification specification);

    Specification findOne(Long id);

    void update(Specification specification);

    void delete(Long[] ids);

    List<Map> selectSpecList();
}
